ui/src/types.js

```diff
@@ -1,3 +1,13 @@
 //@flow
+import { V1Node } from '@kubernetes/client-node';
```

Comment:
I think we can directly use `V1NodeList` to type the return of `useNodes()`
import { V1NodeList } from '@kubernetes/client-node';